modules/log6x-configuring-otlp-output.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.0/log6x-clf.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log6x-configuring-otlp-output_{context}"]
+= Configuring OTLP output
+
+The OpenTelemetry Protocol (OTLP) output allows the cluster administrator to collect and forward logs to OTLP receivers. The OTLP output uses the protocol defined by the https://opentelemetry.io/docs/specs/otlp/[OpenTelemetry Observability Framework] to send data over HTTP with JSON encoding.
+
+:FeatureName: The OTLP output log forwarder
+include::snippets/technology-preview.adoc[]
+
+.Procedure
+
+* To enable OTLP, create or edit the `ClusterLogForwarder` CR using the template below:
```

Comment:
nit but maybe "a ClusterLogForwarder" since there can be more then one...

Additionally I might say something like....

create or edit a ClusterLogForwarder, adding the annotation like below to enable forwarding using OTLP